packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: any;
+    property?: Property;
+    input?: boolean;
```

Comment:
Same question here, how is this set in Builder and what is it used for?

Reply:
This is getting set via code if the field is calculated one then the input will be false otherwise will be true.

Reply:
This will render the value inside the field and the field will not be readOnly. I'm not sure whether to add this to the type definition but the logic in the code is like this `input: !property.formula,`

Reply:
@rverma3 Can you comment on what `input` is for?

Reply:
Input prop removed.